sim/common/sod/item_effects/phase_3.go

```diff
@@ -117,5 +119,25 @@ func init() {
 		})
 	})
 
+	// Serpent's Striker
+	itemhelpers.CreateWeaponProcSpell(SerpentsStriker, "Serpent's Striker", 5.0, func(character *core.Character) *core.Spell {
```

Comment:
Where did you find the 5% proc chance? Just want to verify it and I didn't see it on Wowhead

Reply:
I didn't, I don't think we have any way to know it until the item is live but I want to get it into the sim so this is modelled after Mekkatorque which is mechanically a very similar item.

Reply:
Ok that works for me for now. I'll leave a note to double check the proc chance